bindings/ros/aditof_roscpp/include/aditof_roscpp/generic_msg.h

```diff
@@ -0,0 +1,11 @@
+#ifndef GENERIC_MSG_H
+#define GENERIC_MSG_H
+
+#include <aditof/camera.h>
+
+class GenericMsg {
```

Comment:
Just naming it 'Message' is generic enough.
Since it is a message from aditof sensor why not name this class accordingly?